packages/help-center/src/components/notices.tsx

```diff
@@ -1,28 +1,19 @@
-import { localizeUrl } from '@automattic/i18n-utils';
 import { Button } from '@wordpress/components';
 import { createInterpolateElement } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
 import { useNavigate } from 'react-router-dom';
 import './notices.scss';
 
-export const NewThirdPartyCookiesNotice: React.FC = () => {
+export const BlockedZendeskNotice: React.FC = () => {
 	return (
 		<div className="help-center__notice cookie-warning">
 			<p>
-				<strong>{ __( 'Enable cookies to get support.', __i18n_text_domain__ ) }</strong>
+				<strong>{ __( 'Live support is unavailable.', __i18n_text_domain__ ) }</strong>
```

Comment:
Is it always live support? Some people might not be eligible for live support, right?